example-get-started/generate.sh

```diff
@@ -79,15 +79,16 @@ dvc push
 
 rm data/data.xml data/data.xml.dvc
 dvc import https://github.com/iterative/dataset-registry \
-  get-started/data.xml -o data/data.xml \
+  get-started/data.xml -o data/data.xml --rev 95d720c467496ea6c15dd2c5d5ad48bbb631d8b1 \
```

Comment:
Could we add some tags in the dataset-registry and use them as `rev`? 

Would be a great signal to have semantic meaning in the table (after https://github.com/iterative/studio/issues/2682 is implemented)

Reply:
yes, this is until https://github.com/iterative/dataset-registry/pull/28, and there are few other places I'll revert in the PR